### PR TITLE
minor things

### PR DESCRIPTION
**Pull request by triphora**
* make fabric.mod.json better
* tidy build.yml
* MIT licence
* don't use modid as your mod ID
* icon

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ plugins {
 sourceCompatibility = JavaVersion.VERSION_1_8
 targetCompatibility = JavaVersion.VERSION_1_8
 
-archivesBaseName = project.archives_base_name
+archivesBaseName = 'Empire-Minecraft-Utilities-1.16.x'//project.archives_base_name
```
Comment (MrFrydae) [inline build.gradle:9]: What does this mean exactly? Would the x get replaced by the current revision of mc?

File: src/main/resources/fabric.mod.json [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
   "schemaVersion": 1,
-  "id": "modid",
+  "id": "emcutils",
   "version": "${version}",
 
   "name": "Empire Minecraft Utilities",
-  "description": "This is an example description! Tell everyone what your mod is about!",
+  "description": "A collection of various utilities for the Empire Minecraft server.",
   "authors": [
-    "Me!"
```
Comment (MrFrydae) [inline src/main/resources/fabric.mod.json:9]: But... it is me :(

File: build.gradle (hunk 1 above)
Comment (triphora) [reply build.gradle:9]: No, you would have to replace the version number when/if this is updated to 1.17+. Just allows it to follow the naming conventions of other mods.

File: src/main/resources/fabric.mod.json (hunk 2 above)
Comment (triphora) [reply src/main/resources/fabric.mod.json:9]: Yeah, but... 
do you really want this?

Comment (MrFrydae) [reply src/main/resources/fabric.mod.json:9]: I admit defeat

File: build.gradle (hunk 1 above)
Comment (MrFrydae) [reply build.gradle:9]: Gotcha

**Review by MrFrydae — COMMENTED** [review]
Looks good fam

**Pull request merged**